src/modules/data/views/Messages/Messages.tsx

```diff
@@ -14,31 +14,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import React, { useState, useContext, useEffect } from 'react';
-import { Grid, Typography, Box, Button } from '@mui/material';
+import { Box, Button, Grid, Typography } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
-import { IMessage, IHistory } from '../../../../core/interfaces';
-import { MessageDetails } from './MessageDetails';
-import { ApplicationContext } from '../../../../core/contexts/ApplicationContext';
-import { DataViewSwitch } from '../../../../core/components/DataViewSwitch';
+import React, { useContext, useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
-import { MessageTimeline } from './MessageTimeline';
-import { MessageList } from './MessageList';
-import { FilterModal } from '../../../../core/components/FilterModal';
-import { NamespaceContext } from '../../../../core/contexts/NamespaceContext';
-import { useDataTranslation } from '../../registration';
+import { ArrayParam, useQueryParam, withDefault } from 'use-query-params';
+import { DataViewSwitch } from '../../../../core/components/DataViewSwitch';
+import { DatePicker } from '../../../../core/components/DatePicker';
 import { FilterDisplay } from '../../../../core/components/FilterDisplay';
-import { ArrayParam, withDefault, useQueryParam } from 'use-query-params';
+import { FilterModal } from '../../../../core/components/FilterModal';
+import { ApplicationContext } from '../../../../core/contexts/ApplicationContext';
+import { IHistory, IMessage } from '../../../../core/interfaces';
 import { filterOperators } from '../../../../core/utils';
-import { DatePicker } from '../../../../core/components/DatePicker';
+import { useDataTranslation } from '../../registration';
+import { MessageList } from './MessageList';
+import { MessageTimeline } from './MessageTimeline';
 
 export const Messages: () => JSX.Element = () => {
```

Comment:
Does the histogram endpoint support messages? Hoping we could have those awesome charts in the messages view.

Reply:
Good idea, added it for messages and transactions list views

Reply:
### Transactions
![tx](https://user-images.githubusercontent.com/17093033/145495362-3a870887-b109-4327-817f-a46f60aed1dd.png)

### Messages
![msgs](https://user-images.githubusercontent.com/17093033/145495428-f04d6829-e11a-4dbb-bc1b-f55083f7e278.png)